src/common/config.cc

```diff
@@ -473,6 +473,31 @@ void md_config_t::parse_env(unsigned entity_type,
       _set_val(values, tracker, dir, *o, CONF_ENV, &err);
     }
   }
+
+  // Apply pod memory limits:
+  //
+  // There are two types of resource requests: `limits` and `requests`.
+  //
+  // - Requests: Used by the K8s scheduler to determine on which nodes to
+  //   schedule the pods. This helps spread the pods to different nodes. This
+  //   value should be conservative in order to make sure all the pods are
+  //   schedulable.
+  // - Limits: At runtime, K8s will use the limits to see if the pod is using
+  //   too many resources. In that case, the pod will be killed/restarted
+  //   automatically if the pod goes over the limit. This should be higher than
+  //   what is specified for requests (potentially much higher). 
+  //
+  // Here are the documented best practices: https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#motivation-for-cpu-requests-and-limits
+  //
+  // When the operator creates the CephCluster CR, it will need to generate the
+  // desired requests and limits. As long as we are conservative in our choice
+  // for requests and generous with the limits we should be in a good place to
+  // get started.
+  //
+  // The support in Rook is already there for applying the limits as seen in these links.
+  //
+  // Rook docs on the resource requests and limits: https://rook.io/docs/rook/v1.0/ceph-cluster-crd.html#cluster-wide-resources-configuration-settings
```

Comment:
This link is to the 1.0 release docs so it will get stale over time. However, if we change it to a master link we would be at risk of the link not working if something changes so maybe it's better to keep the 1.0 link.

Reply:
I will keep the 1.0 link I guess. :)